cloudhub-2/modules/ROOT/pages/_partials/deployment.adoc

```diff
@@ -100,23 +100,22 @@ To configure additional log levels and categories to include in logs:
 
 = Enable Trace Data Collection
 
-Traces enable you to understand the execution path of a transaction in MuleSoft ecosystem by having full visibility into the execution path of API calls or application transactions. This feature exposes tracing information in OpenTelemetry standard from Mule runtime into the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter].
+[NOTE]
+include::reuse::partial$billing/pricing.adoc[tag=featureAdvReq]
 
-You can enable trace data collection for your CloudHub 2.0 applications running on Mule runtime version 4.6 or later. 
+Traces enable you to understand the path of a transaction in the MuleSoft ecosystem by providing full visibility into the execution path of API calls or application transactions. 
+When you enable trace data collection for an app, Anypoint Platform collects and exposes tracing information from Mule Runtime in the OpenTelemetry standard, enabling you to export it using the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter]. 
```

Comment:
```suggestion
When you enable trace data collection for an app, Anypoint Platform collects and exposes tracing information from Mule runtime engine in the OpenTelemetry standard, enabling you to export it using the Anypoint Monitoring xref:monitoring::telemetry-exporter.adoc[Telemetry Exporter]. 
```

Reply:
From the Term Presentation Reference: Mule (after spelling out "Mule runtime engine" at first use per topic set)